### ISSUE TITLE
Is HFE still actively maintained?

### ISSUE THREAD

**Issue by stevesobol** [scottlerch/HostsFileEditor#32]
Doesn't look like it is.

I have some new features I might want to add, and I'm deciding between "send in pull requests" and "just fork and do my own thing".

**Comment by dimbslmh** [comment]
https://github.com/scottlerch/HostsFileEditor/pull/33 looks like @scottlerch plans to convert this project to a self-contained .Net Core app

**Comment by scottlerch** [comment]
Yes — [v1.3.0](https://github.com/scottlerch/HostsFileEditor/releases/tag/v1.3.0) just shipped: a .NET 10 rewrite with a new modern (WinUI) edition heading to the Microsoft Store, alongside the classic build. Contributions are welcome. Closing.
